app/build.gradle.kts

```diff
@@ -86,6 +86,7 @@ dependencies {
     implementation(project(":core:ui"))
     implementation(project(":core:designsystem"))
     implementation(project(":core:navigation"))
+    implementation(project(":core:model"))
```

Comment:
could this dependency be causing the build issues? @alexvanyo @JolandaVerhoef - any ideas on how to repro/fix this locally?